app/main.py

```diff
@@ -16,4 +16,20 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+massive = [lucky_number, pi, one_is_a_prime_number, name, my_favourite_films,
+           profile_info, marks, collection_of_coins]
+
+
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
```

Comment:
you can do it simply like this

```suggestion
sorted_variables = {
    "mutable": [],
    "immutable": [lucky_number, pi ....]
}
```